libs/vue-plugin/package.json

```diff
@@ -13,6 +13,12 @@
     "@angular-devkit/core": "~9.1.0",
     "@angular-devkit/schematics": "~9.1.0",
     "@vue/cli-shared-utils": "~4.3.0",
-    "rxjs": "6.5.4"
+    "less": "^3.0.4",
+    "less-loader": "^5.0.0",
+    "rxjs": "6.5.4",
+    "sass": "^1.26.5",
```

Comment:
Change version to `^1.26.3`. Based on version of Vue CLI we install (i.e. `4.3.1`).

https://github.com/vuejs/vue-cli/blob/v4.3.1/packages/%40vue/cli-service/generator/index.js#L27